src/containers/ProjectDashboardContainer.jsx

```diff
@@ -20,18 +23,40 @@ const propTypes = {
 };
 
 class ProjectDashboardContainer extends Component {
-
+  constructor() {
+    super();
+    this.handleSelect = this.handleSelect.bind(this);
+    this.state = {
+      translationLanguage: '',
+    };
+  }
   componentDidMount() {
     const { actions } = this.props;
     actions.fetchProject(this.props.params.project_id);
   }
 
+  handleSelect(event) {
+    this.setState({
+      translationLanguage: event.target.textContent,
+    });
+  }
+
   render() {
     const project = this.props.project.data;
     const { fieldguides, pages, workflows, tutorials } = this.props.project;
+    const renderDropdownItems = languages.map((item) => {
+      return (
+        <DropdownItem key={item.code} onSelect={this.handleSelect}>
```

Comment:
Should the selected item be indicated, in the same way that HTML select menus use the `selected` attribute for the selected option?